cfme/tests/services/test_catalog.py

```diff
@@ -31,12 +32,20 @@ def test_catalog_duplicate_name():
     cat.delete()
 
 
+@pytest.mark.meta(blockers=[BZ(1460891, forced_streams=['5.8', '5.9', 'upstream'])])
 @pytest.mark.sauce
 def test_permissions_catalog_add(appliance):
     """ Tests that a catalog can be added only with the right permissions"""
     cat = Catalog(name=fauxfactory.gen_alphanumeric(),
                   description="my catalog")
 
-    tac.single_task_permission_test(appliance,
-                                    [['Everything', 'Services', 'Catalogs Explorer', 'Catalogs']],
-                                    {'Add Catalog': cat.create})
+    test_product_features = [['Everything', 'Services', 'Catalogs Explorer', 'Catalogs']]
+
+    # Since we try to create the catalog with the same name, we obliged to delete it after creation
+    # in order to avoid "Name has already been taken" error which makes this test "blind" to the
+    # fact, that disallowed action actually can be performed.
+    # TODO: remove this workaround with "lambda"
+    test_actions = {'Add Catalog': lambda _: cat.create(),
+                    'Delete Catalog': lambda _: cat.delete()}
```

Comment:
this workaround totally depends on existing of [this function](https://github.com/ManageIQ/integration_tests/pull/6500/files#diff-b61c9b90f0aab89bdb8a251987a47c93R663)